### ISSUE TITLE
wait for over 30 minutes on pre-rendering

### ISSUE THREAD

**Issue by StarsTesla** [zzzyuqing/DreamMat#17]
我用了你的Docker image，然后下载了ckpt和blender，指定了blender path，运行起来后，这个prerender等了30多分钟了还没好，这是为啥？
我在4090上跑的：
![image](https://github.com/zzzyuqing/DreamMat/assets/45264423/0a0c0264-c703-4672-90a5-ec0861fae7e2)

**Comment by zzzyuqing** [comment]
During the pre-rendering phase, there are a large number of read and write operations, which are quite related to CPU performance (15min on i9-14900k). If the process appears to be too slow, increasing the number of threads could be a solution.

**Comment by StarsTesla** [comment]
my server got like 128 process, I dont where is wrong

**Comment by XZYW7** [comment]
You can check if there is any wrong in Blender rendering process following the comment in this [issue](https://github.com/zzzyuqing/DreamMat/issues/10#issuecomment-2159729925). 
And you can also check the actual utilized threads number by output "bpy.context.scene.render.threads" in blender script which should be the max usable threads num in default.

**Comment by StarsTesla** [comment]
@XZYW7 It may be the problem of blender, so I run on my local machine, after almost 10 minutes, the rendering is done, 
![image](https://github.com/user-attachments/assets/10a44898-13b4-4d56-969b-089a1ff2c004)
Then stucked........... WTF?

**Comment by XZYW7** [comment]
sry I don't understand what the stucked mean in the image. Did it continue later?

**Comment by StarsTesla** [comment]
@XZYW7 No, I wait for over 20minutes, and nothing shows up, this is very unfriendly

**Comment by XZYW7** [comment]
The next stage should be loading sd model. I think it is stucked in downloading according to the warning in file_dowload.py.

**Comment by jloveric** [comment]
I was using an A800 and it takes about 40ish minutes the first time. Just have to be patient. I get the impression that would be about 10 minutes on a 4090 though.
